frontend/src/employee-frontend/components/unit/tab-placement-proposals/PlacementProposalRow.tsx

```diff
@@ -141,23 +198,32 @@ export default React.memo(function PlacementProposalRow({
         </Td>
         <Td data-qa="application-link">
           <CenteredDiv>
-            <a
-              href={`${getEmployeeUrlPrefix()}/employee/applications/${
-                placementPlan.applicationId
-              }`}
-              target="_blank"
-              rel="noreferrer"
-            >
+            {confirmationState !== 'REJECTED_BY_CITIZEN' ? (
+              <a
+                href={`${getEmployeeUrlPrefix()}/employee/applications/${
+                  placementPlan.applicationId
+                }`}
+                target="_blank"
+                rel="noreferrer"
+              >
+                <IconButton
+                  onClick={() => undefined}
+                  icon={faFileAlt}
+                  altText={i18n.personProfile.application.open}
+                />
+              </a>
+            ) : (
               <IconButton
                 onClick={() => undefined}
```

Comment:
I'm no UX expert but it would feel more intuitive to have an empty cell instead of a button that cannot ever be interacted with.